### DISCUSSION TITLE
FS22 no carga  ultima version coursePlay

### DISCUSSION THREAD

**Discussion by lopez65** [Courseplay/Courseplay_FS22#2763] [Q&A] [answered]
Hello
I just downloaded CoursePlay for Farming Simulator v.1.8.2.0 (29243 72209) 22 Pc, when I start the game it doesn't load the mod, it loads all the mods I have installed except AutoDrive and CoursePlay. The previous version worked for me.
thanx.

**Comment by Tensuko** [comment, marked as answer]
Becouse your FS Version is completly outdated.
Actual Version is 1.12.0.0! You have to update your game.

**Reply by lopez65** [reply]
thanx for the info
